gemrb/core/Scriptable/Scriptable.cpp

```diff
@@ -2143,49 +2145,91 @@ unsigned char Movable::GetNextFace()
 
 // returns whether we made all pending steps (so, false if we must be called again this tick)
 // we can't just do them all here because the caller might have to update searchmap etc
-bool Movable::DoStep(unsigned int walk_speed, ieDword time)
-{
-	if (!path) {
-		return true;
+bool Movable::DoStep(unsigned int walkAnimFrames, ieDword time) {
```

Comment:
walkAnimFrames is not the best name, since it's called with a transformed number, not the count of walk Anim Frames, but I also don't have any better suggestions. It's used as a time and a distance in this function. :confused: